src/QuoteBox.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import './App.css';
+
+class QuoteBox extends Component {
+	// constructor(props) {
+	// 	super(props)
+	// }
+	
+  render() {
+
+    const quote = this.props.quote;
```

Comment:
FYI you can also write `const { quote } = this.props`
but this is totally valid and good code
(just a tip)